packages/flutter/lib/src/services/asset_manifest.dart

```diff
@@ -24,6 +22,12 @@ abstract class AssetManifest {
     return bundle.loadStructuredBinaryData(_kAssetManifestFilename, _AssetManifestBin.fromStandardMessageCodecMessage);
```

Comment:
I think this is the place where the conditionally imported `loadAssetManifest` should be used, instead of where it's now in the `obtainKey` method of the `image_resolution.dart`.

This line causes the `_kAssetManifestFilename` to be downloaded. Trying to prep a small fix.

Reply:
Pushed fix.